### PR TITLE
Add FigmaExportOptions

### PR DESCRIPTION
**Pull request by Alexander-Ignition**
Extracted common options from the commands 

[Defining Commands and Subcommands](https://github.com/apple/swift-argument-parser/blob/main/Documentation/03%20Commands%20and%20Subcommands.md)

### CODE REVIEW

File: Sources/FigmaExport/Input/FigmaExportOptions.swift [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    static let input = "figma-export.yaml"
+
+    @Option(name: .shortAndLong, help: "An input YAML file with figma and platform properties.")
+    var input: String = Self.input
+
+    var accessToken: String = ""
+
+    var params = Params.placeholder
+
+    var unpack: (String, Params) {
```
Comment (subdan) [inline Sources/FigmaExport/Input/FigmaExportOptions.swift:15]: Why did you add `unpack` property? We can get access to `accessToken` and `params` directly from the `ExportImages`, `ExportColors`, etc.

File: Sources/FigmaExport/Input/FigmaExportOptions.swift [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+struct FigmaExportOptions: ParsableArguments {
+    static let input = "figma-export.yaml"
+
+    @Option(name: .shortAndLong, help: "An input YAML file with figma and platform properties.")
+    var input: String = Self.input
+
+    var accessToken: String = ""
+
+    var params = Params.placeholder
```
[the remark this answers is not in the document]
Comment (subdan) [reply Sources/FigmaExport/Input/FigmaExportOptions.swift:13]: I think it is safe because we will use this property after validation.

File: Sources/FigmaExport/Input/FigmaExportOptions.swift (hunk 1 above)
Comment (Alexander-Ignition) [reply Sources/FigmaExport/Input/FigmaExportOptions.swift:15]: I made `unpack` so as not to change the rest of the code in the commands. I'll redo it

**Review by subdan — APPROVED** [review]

**Comment by subdan** [comment]
Good work. Thanks!
